circle.yml

```diff
@@ -204,6 +212,8 @@ jobs:
   darwin_smoke:
     macos:
       xcode: 13.4.1 # Mac OS 12.6.1, see https://circleci.com/docs/using-macos/
+    environment:
+      GOPHERJS_EXPERIMENT: generics
```

Comment:
Do we want to run our tests (or maybe a subset) both with and without generics support, to ensure we don't break non-generics support during this transition?

Reply:
Running all tests seems prohibitively expensive, considering that they ballooned up to 30m again. But it may be worth having a smoke test of some sort. What do you think would be a reasonable subset?